.travis.yml

```diff
@@ -0,0 +1,15 @@
+env:
+global:
+- CC_TEST_REPORTER_ID=d570e2625aaffa20e7828eaddfbec748c525e866b6d413472bc26e188cb2bc6a
+language: node_js
+node_js:
+- "stable"
+cache:
+directories:
+- "node_modules"
+before_script:
+- curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
```

Comment:
@sengayire why are you sending the report before the script executes?

Reply:
I am going to fix it